### PR TITLE
resources: added governing-body-classification-code

### PR DESCRIPTION
**Pull request by LBONTEN**
For the data quality page, we needed the classification of governing bodies as well as for administrative units. 
Though apparently the classifications for governing bodies were missing. So now I added them :)

### CODE REVIEW

**Review by aliokan — CHANGES_REQUESTED** [review]
@LBONTEN Can you add the `classification` relationship in `governing-body` resource?

**Comment by LBONTEN** [comment]
> @LBONTEN Can you add the `classification` relationship in `governing-body` resource?

yeah I forgot that 😅 fixed it in the next commit (https://github.com/lblod/app-burgernabije-besluitendatabank/pull/41/commits/4b71f0d1d57efbea3bd1e99128e5ae24941596d1)

File: config/resources/besluit-domain-en.lisp [hunk 1]
```diff
@@ -84,8 +84,8 @@
                 (:start-date :date ,(s-prefix "mandaat:bindingStart")))
   :has-one `((administrative-unit :via ,(s-prefix "besluit:bestuurt")
                                   :as "administrative-unit")
-  ;;            (bestuursorgaan-classificatie-code :via ,(s-prefix "besluit:classificatie")
-  ;;                                               :as "classificatie")
+             (bestuursorgaan-classificatie-code :via ,(s-prefix "besluit:classificatie")
```
Comment (aliokan) [inline config/resources/besluit-domain-en.lisp:87]: @LBONTEN **Blocker:** Can you translate `governing-body-classification-code` and `classification`

Comment (LBONTEN) [reply config/resources/besluit-domain-en.lisp:87]: 👍

File: config/resources/besluit-domain-en.lisp [hunk 2]
```diff
@@ -84,8 +84,8 @@
                 (:start-date :date ,(s-prefix "mandaat:bindingStart")))
   :has-one `((administrative-unit :via ,(s-prefix "besluit:bestuurt")
                                   :as "administrative-unit")
-  ;;            (bestuursorgaan-classificatie-code :via ,(s-prefix "besluit:classificatie")
-  ;;                                               :as "classificatie")
+             (governing-body-classification-code :via ,(s-prefix "besluit:classificatie")
+                                                :as "classificatie")
```
Comment (aliokan) [inline config/resources/besluit-domain-en.lisp:88]: @LBONTEN Almost ^^

```suggestion
                                                :as "classification")
```

**Review by aliokan — APPROVED** [review]
LGTM! :+1:

**Pull request merged**